tests/test_models.py

```diff
@@ -550,6 +666,18 @@ def test_draft_status_for_brief_response_with_no_submitted_at(self):
         brief_response = BriefResponse(created_at=datetime.utcnow())
         assert brief_response.status == 'draft'
 
+    def test_awarded_status_for_brief_response_has_been_awarded(self):
+        with self.app.app_context():
+            brief = Brief.query.get(self.brief_id)
+            brief_response = BriefResponse(
+                brief=brief,
+                submitted_at=datetime.utcnow(),
+                award_details={'confirmed': 'details'},
+                awarded_at=datetime(2016, 1, 1)
+            )
+
```

Comment:
Cool - have pushed a commit to fix (re-ordering the case statement), which also includes a test for the `pending-awarded` status.